apps/src/codebridge/FileTabs/styles/fileTabs.module.scss

```diff
@@ -5,11 +5,11 @@
   justify-content: flex-start;
   gap: 6px;
   align-items: flex-end;
-  overflow-x: scroll;
+  overflow-x: auto;
   max-width: 100%;
   background-color: $light_gray_900;
-  scrollbar-width: none;
-  min-height: 32px;
+  scrollbar-width: thin;
+  box-sizing: border-box;
```

Comment:
When you hover on the scrollbar, the bar covers up a lot of the filename. 
![Screenshot 2025-04-15 at 3 57 02 PM](https://github.com/user-attachments/assets/d42430f4-109f-42b1-a2ee-0ad95460cc9c)

Maybe add `padding-bottom` to `.fileTab`?

Reply:
huh this does not happen for me. What browser/version are you using?

Reply:
Chrome v 135.0.7049.85

Reply:
Discussed offline--this turns out to be universal behavior if you have the mac setting that scroll bars only show up when you are scrolling. Since this behavior also exists for the editor, and everything is still functional, we will keep it as-is.

Reply:
Discussed offline - scrollbar issue is universal with the 'Automatically based on mouse or trackpad' or 'When scrolling' setting for Macbooks - happens in the editor as well - the browser doesn't account for the scrollbar. Adding padding was already considered and declined at https://codedotorg.slack.com/archives/D03KG9BUJN6/p1744836141197559. This update is definitely an improvement so agreed to keep as-is. Thanks for digging into this Molly!